lib/collections/schemas/helpers.js

```diff
@@ -55,3 +56,30 @@ export function schemaIdAutoValue() {
   }
   return this.unset();
 }
+
+/**
+ * shopDefaultCountry
+ * @summary used for schema injection autoValue
+ * @example autoValue: shopDefaultCountry
+ * @return {String} returns country value from default shop
+ */
+export function shopDefaultCountry() {
+  // Check to see if this is client or server, and the type of update being performed
+  if (this.isSet && Meteor.isServer) {
+    return this.value;
+  } else if (Meteor.isServer && !this.isUpdate || Meteor.isClient && this.isInsert) {
+    // Find the current shop
+    const shop = Shops.findOne({
+      _id: Reaction.getShopId()
+    });
+
+    // Find the current shops primary address
+    if (shop && shop.addressBook && shop.addressBook[0]) {
```

Comment:
Not sure how much work this would really be,  but why we don't extend this a little more, with the goal of removing `[0]` dependencies, and remove the need refactor this again (so soon)? One approach that seems easy, could be to just set and check `isShippingDefault` to know the origination country?